WalletWasabi.Fluent/ViewModels/MainViewModel.cs

```diff
@@ -128,6 +136,19 @@ public MainViewModel()
 				}
 			});
 
+		this.WhenAnyValue(x => x.MainScreen.CurrentPage)
```

Comment:
Another way to express this is:

```
		this.WhenAnyValue(x => x.MainScreen.CurrentPage)
			.Select(cp => cp as WalletViewModel)
			.Subscribe(w => CurrentWallet = w);
```
Just in case, should you consider it's more readable :)

Reply:
code was a leftover and removed.